e2e/journey-app/main.ts

```diff
@@ -23,6 +23,8 @@ const searchParams = new URLSearchParams(qs);
 
 const config = serverConfigs[searchParams.get('clientId') || 'basic'];
 
+const tree = searchParams.get('tree') ?? 'UsernamePassword';
```

Comment:
I'm using `journey` and not `tree` in all the tests I've migrated over. Let's stick with that since "tree" is an outdated term.

Reply:
`journey` conflicts with the variable `journey` we can use a variation of `journey` if you prefer that?